docker/scripts/update-docker.sh

```diff
@@ -12,7 +12,7 @@ echo "Initializing dgraph schema..."
 php artisan schema:init --yes
 
 echo "Creating admin user"
-php artisan admin_user:create
+php artisan user:create {}
```

Comment:
Is the `{}` intentional here? I'm wondering if we should be using the user check here:

```suggestion
php artisan user:create --userCheck
```

Reply:
Yep, that is what is meant to be there - thanks :)